ada/language/ast.py

```diff
@@ -9395,15 +9395,14 @@ def primitive_subp_of():
             Entity.returns._.singleton
         ))
 
-        return Self.unique_nodes(types.map(
```

Comment:
I had to read the code to verify that you were not modifying anything. Putting that in a "Minor: reformatting" commit would be nice :slightly_smiling_face:

Reply:
My bad, you are indeed modifying something :man_facepalming: